contributor-docs/adrs/adr-017-css.md

```diff
@@ -73,16 +73,16 @@ The other reasons for change are to utilise css variables for theming and improv
 Code example:
 
 ```jsx
-import classNames from './ActionList.module.css'
 import clsx from 'clsx'
+import styles from './ActionList.module.css'
```

Comment:
pretty silly nitpick but, can we call the variable `classNames` instead of styles because it contains classNames not the actual styles

Reply:
yeah all good! probably just me. too many "classNames", that it's hard to quickly scan a file and know which are imported styles, which are className's in JSX and which are forwarded className props. 

but not offended one way or another! i'll rename 🎉